style_guides/frontend-review-checklist.md

```diff
@@ -0,0 +1,33 @@
+# Frontend Review Checklist
+
+## High level checks
+
+- [ ] The pull request adds value and the impact of the change is in line with [Frontend Style Guide](https://github.com/grafana/grafana/blob/master/style_guides/frontend.md).
+- [ ] The pull request works the way it says it should do.
+- [ ] The pull request does not increase the Angular code base.
+  > We are in the process of migrating to React so any increment of Angular code is generally discouraged from. (there are a few exceptions)
+- [ ] The pull request closes one issue if possible and does not fix unrelated issues within the same pull request.
+- [ ] The pull request contains necessary tests.
+
+## Low level checks
+
+- [ ] The pull request contains a title that explains the PR.
+- [ ] The pull request contains necessary link(s) to issue(s).
+- [ ] The pull request contains commits with commit messages that are small and understandable.
+- [ ] The pull request does not contain magic strings or numbers that could be replaced with an `Enum` or `const` instead.
+- [ ] The pull request does not increase the number of `implicit any` errors.
```

Comment:
As a follow up issue we could try to introduce some pre-commit check that would warn/error on increasing number of `implicity any` errors

Reply:
I think @bergquist had an excellent idea on how to solve this by breaking the CI-build if some type of errors reach a certain limit. But we should use that logic in pre-commit check as well.